liferay-echo/liferay-portal
# [TS] LPS-162468 Update UI for displaying current sorting type

**Motivation**: 
[LPP-46398](https://issues.liferay.com/browse/LPP-46398): Asset Publisher's filtering by category, and display order by descending Publish Date is not working properly

This PR is fixing the display of the sorting icon in the UI.

**Step to reproduce**: 

1. Start up a Liferay DXP 7.4 U26 instance connected to a MySQL database

2. Sign in as admin

3. In the default Liferay DXP site, go to the side menu > Categorization > Categories

4. Create a new vocabulary called "VocabTest"

5. Go into "VocabTest" and create a category called "webinar"

6. Create 5 Basic Web Content articles (e.g. TestArticle 1, TestArticle 2, ... TestArticle 5) with each of them having the "webinar" category assigned to it (i.e., In the Web Content editor, click on Categorization > VocabTest > Select – Select "webinar")

7. In the Liferay DXP site, add a widget page

8. In that new widget page, add an Asset publisher Widget

9. Click on tricolon icon for the Asset Publisher widget > Configuration, and under Asset Selection, configure it as follows:
      Asset Selection: Dynamic
      Source
      Asset Type: Web Content Article
      Web Content Article Structures: Any
      Filter
      Rules Contains All of the following Categories
      Categories: webinar
      Ordering
      Order by Publish Date And Then Title
      Make sure both arrows are set to Descending order
10. Go back to the Widget page, and assert that the ordering is ignored with the first article created being on top, instead of the latest article (which is what we'd expect of Descending order)

11. Edit the bottom article (i.e., TestArticle 5) and under Schedule change the Display date to around 5 minutes from now, then publish.

12. Wait around 5 minutes, and check the widget page containing our Asset Publisher. Note: If TestArticle 5 is not yet published, then wait a little longer.

**Actual Behavior**
"TestArticle 5" still remains at the bottom of the Asset Publisher, even though Asset Publisher is configured to show items in Descending order.

**Expected Behavior**
The future display date's article should now come on top as Article's display date gets converted into Publish Date in asset entry. Basically, order by descending is not working properly in Asset Publisher.

cc: @hongvo2308
